requirements.txt

```diff
@@ -1,2 +1,2 @@
-sphinx==1.4.9
-git+https://github.com/StackStorm/sphinx-autobuild.git@develop
+sphinx
```

Comment:
Yeah, that's a good point. I'll make that change. I've also done some testing and it looks good, but I want to do a little more testing here on a 'clean' system to verify it, then we can merge it.